packages/react/src/components/calendar/use-calendar-picker.ts

```diff
@@ -171,8 +170,7 @@ export const useCalendarPicker = <T extends UseCalendarProps<any>>(
     hiddenOutsideDays,
     holidays,
     inputFormat = "YYYY/MM/DD",
-    isLazy,
-    isOpen,
+    lazy,
     lazyBehavior,
     locale = themeLocale ?? defaultLocale,
     matchWidth,
```

Comment:
When I remove the deprecated props `isLazy` (from popover package) in the hook, I realized that the `lazy` prop is not passed in and used, so I added back here.